example/data-migration1-tf/main.tf

```diff
@@ -0,0 +1,23 @@
+data "terraform_remote_state" "data_persistence" {
+  backend   = "s3"
+  config    = var.data_persistence_remote_state_config
+  workspace = terraform.workspace
+}
+
+module "data_migration1" {
```

Comment:
If anyone has a better suggestion for the name of migration for collection data (and possibly later providers) that has to run **before** re-deploying Cumulus, I'm all ears

Reply:
cumulus-dynamo-to-rds-migration?

Reply:
It's not going to be the only dynamo-to-rds migration though. As I understood, the plan was to have this first migration for data that granules reference (e.g. collections, providers), which has to run **before** deploying the version of Cumulus that does parallel writes. Then there would be a second data migration that is included with the Cumulus deployment to migrate existing granules/files data. So I don't know how to distinguish between them beyond calling this migration number 1 and the second migration number 2.

Reply:
Right, sorry, I mean dynamo-to-rds-migration{n}.   It's not a great suggestion, frankly, so we can leave that be.